src/app/shared/menu/export/export-google-sheets.service.spec.ts

```diff
@@ -0,0 +1,19 @@
+import { ExportGoogleSheetsService } from './export-google-sheets.service';
+
+declare const gapi: any;
+
+describe('ExportGoogleSheetsService', () => {
```

Comment:
there really isn't much I to test here except that this ignores requests when the google api is not loaded, and it does not get loaded in the test env